libs/feature/map/src/lib/utils/map-utils.service.ts

```diff
@@ -193,10 +193,10 @@ export class MapUtilsService {
 
 export function dragPanCondition(
   this: DragPan,
-  event: MapBrowserEvent<UIEvent>
+  event: MapBrowserEvent<PointerEvent>
 ) {
   const dragPanCondition =
-    this.getPointerCount() === 2 || platformModifierKeyOnly(event)
+    this.getPointerCount() === 2 || event.originalEvent.pointerType === 'mouse'
```

Comment:
Wondering how we could only listen to the last event for the 2 finger condition here, as on mobile a two finger touch dispatches a first `pointerdown` event with one `activePointer` and only a second `pointerdown` event with two `activePointers`.

Reply:
Also implemented in the following commit. Thanks for the help @jahow !